playbooks/generic/clean-obsolet-service.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Clean up obsolet service configuration
+  hosts: all
+  vars:
+    obsolet_service:
+    obsolet_service_os_name:
+    obsolet_containers_volumes:
+    obsolet_service_containers:
+    obsolet_service_dir:
+
+    gather_facts: false
+  tasks:
+    - name: "Stop/disable {{ obsolet_service }}-containers"
+      become: true
+      community.docker.docker_container:
+        name: "{{ item }}"
+        state: absent
+        stop_signal: true
+      with_items:
+        - "{{ obsolet_service_containers }}"
+
+    - name: "Stop/disable {{ obsolet_service }}-volumes"
+      become: true
+      community.docker.docker_volume:
+        name: "{{ item }}"
+        state: absent
+      with_items:
+        - "{{ obsolet_service }}"
+
+    - name: "Remove {{ obsolet_service }}-images"
+      become: true
+      community.docker.docker_image:
+         name: "{{ obsolet_service }}"
+         state: absent
+
+    - name: Remove etc directory
+      become: true
+      ansible.builtin.file:
+        path: "/etc/kolla/{{ item }}"
+        state: absent
+      with_items:
+        - "{{ obsolet_service_dir}}"
+
+    - name: Remove log Directory
+      become: true
+      ansible.builtin.file:
+        path: "/var/log/kolla/{{ obsolet_service}}"
+        state: absent
+
+    - name: "Remove {{ obsolet_service }} Loadbalancer Configuration"
+      become: true
+      ansible.builtin.file:
+        path: "/etc/kolla/haproxy/service.d/{{ item }}.cfg"
+        state: absent
+      with_items:
+        - "{{ obsolet_service_dir}}"
```

Comment:
This has to be run on the load balancer nodes.

Reply:
Change to loadbalancer nodes